### DISCUSSION TITLE
Is bref planning to migrate custom runtimes from Lambdas and Layers from Amazon Linux 2 to AL2023?

### DISCUSSION THREAD

**Discussion by remagdoku** [brefphp/bref#1838] [General]
With Amazon Linux 2 EOL being around a corner, I was wondering if in Bref's Road Map is planned to migrate to Amazon Linux 2023.
My organization knows from first hand that it can be hard to migrate from AL2 to AL2023, since Amazon decided to take a change of direction in their development, like not supporting EPEL repositories (https://serverfault.com/questions/1134749/aws-2023-ami-linux-installation-of-epel-release-latest-8), and some others handicaps.

![AmazonLinux2_EOL](https://github.com/brefphp/bref/assets/101346493/b21bfc98-dc24-4c42-b820-632a4fc7f5f7)

**Comment by mnapoli** [comment]
Yes, please see https://github.com/brefphp/aws-lambda-layers/pull/122
